pkg/reconciler/service/test.go

```diff
@@ -2,6 +2,7 @@ package service
 
 import (
 	"context"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler"
```

Comment:
You have some problem with formatting here... It should go below and join other `github.com` imports.